dm_control/locomotion/walkers/assets/dog_v2/muscles.py

```diff
@@ -0,0 +1,249 @@
+lateral = ['m_abdominis_Aponeurosis',
+         'm_depressor_caudae_brevis_L',
+         'm_depressor_caudae_brevis_L2',
+         'm_depressor_caudae_brevis_L2(2)',
+         'm_depressor_caudae_brevis_R',
+         'm_depressor_caudae_brevis_R2',
+         'm_depressor_caudae_brevis_R2(2)',
+         'm_dorsi_Fascia_thoracolumbar_',
+         'm_intercostals_ext_L',
+         'm_intercostals_ext_R',
+         'm_intercostals_int_L',
+         'm_intercostals_int_R',
+         'm_latissimus_dors_L',
+         'm_latissimus_dors_R'
+         'm_levator_caudae_brevis_L',
+         'm_levator_caudae_brevis_L1',
+         'm_levator_caudae_brevis_L2',
+         'm_levator_caudae_brevis_R',
+         'm_levator_caudae_brevis_R1',
+         'm_levator_caudae_brevis_R2',
+         'm_Ligament',
+         'm_longissimus_capitis_L',
+         'm_longissimus_capitis_R(2)',
+         'm_longissimus_lumborum_L',
+         'm_longissimus_lumborum_R',
+         'm_longissimus_thoracis_L',
+         'm_longissimus_thoracis_R',
+         'm_obliquus_ext_abdominis_L',
+         'm_obliquus_ext_abdominis_R',
+         'm_obliquus_int_abdominis_L',
+         'm_obliquus_int_abdominis_R',
+         'm_psoas_major_L',
+         'm_psoas_minor_L',
+         'm_psoas_major_R',
+         'm_psoas_minor_R',
+         'm_rectus_abdominis_R',
+         'm_rectus_abdominis_L',
+         'M_Thoracis_M_Pectoralis_Profundus',
+         'm_transverses_thoracis_L',
+         'm_transverses_thoracis_R',
+         'm_tranversus_abdominis_L',
+         'm_tranversus_abdominis_R']
+
+to_skip = [
+               # 'm_serratus_dorsalis_caudalis_L1',
```

Comment:
bunch of comments and commented-out lines in this file.

either delete or explain why they are here like
```py
# uncomment the lines below for serratus muscles
```
or whatever. 

Also for other comments and commented lines in this file.